test/transforms.jl

```diff
@@ -680,6 +680,49 @@
     @test n1 == n2
   end
 
+  @testset "StdNames" begin
+    names = (:a, Symbol("apple tree"), Symbol("banana tree"))
```

Comment:
I suggest adding more complicated names such as `foo_Bar`, `A b`, ... you need to test the names extensively.

Ideally you will introduce a small table for each kind of name issue. For example, you can introduce a table with names that have underscore, another table with names that have spaces, another table with upper and lower case mixed, etc. This will cover more issues.

Reply:
@juliohm  I know we already discussed this, but don't you think we should also split at underscores? I actually think it would be a reasonable thing to do.

Reply:
@ceferisbarov it depends on the transform, for `:snake` we don't want to split underscores.